api_tests/nodes/views/test_node_logs.py

```diff
@@ -108,8 +108,10 @@ def test_project_add_remove_contributor(self):
     def test_remove_addon(self):
         self.public_project.add_addon('github', auth=self.user_auth)
         assert_equal('addon_added', self.public_project.logs[OSF_LATEST].action)
+        old_log_length = len(list(self.public_project.logs))
         self.public_project.delete_addon('github', auth=self.user_auth)
         assert_equal('addon_removed', self.public_project.logs[OSF_LATEST].action)
+        assert_equal((len(list(self.public_project.logs)) - 1), old_log_length)
```

Comment:
The test asserts that only one log is made, that way if extra logs are made, the test will fail
At least, that is what it is supposed to do